frontend/model/contracts/members/removeMemberSideEffect.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+import sbp from '~/shared/sbp.js'
+
+export default async function removeMemberSideEffect (data) {
+  const rootState = sbp('state/vuex/state')
+  const contracts = rootState.contracts || {}
+
+  if (data.member === rootState.loggedIn.username) {
+    // If this member is re-joining the group, ignore the rest
+    // so the member doesn't remove themself again.
+    if (sbp('okTurtles.data/get', 'JOINING_GROUP')) {
+      return false
```

Comment:
Recommend just `return`, since the return value isn't used by anything, and we don't want to imply it is.